accounts/models.py

```diff
@@ -1,2 +1,30 @@
-# from django.db import models
-# from django.contrib.auth.models import User
+from django.db import models
+from django.contrib.auth.models import User
+# Each user can add, modify or remove course from is list
+# Each course contains coures id(default by Django), name, credinitials and grade
+
+
+class Course (models.Model):
+    user = models.ManyToManyField(User, blank=True)
+    course_name = models.CharField(max_length=50)
+    credinitials = models.IntegerField(max_length=1)
```

Comment:
What is this field for?

Reply:
It is used to calculate the GPA. (sum(grade * credinitials)/ credinitials).

Reply:
Instead of including a field for GPA (which can change every so often), might be better to calculate it from a `grades` (or similar) table when it is needed

Reply:
this is what i want to do. i didnt make filed of GPA only for grades, the credinitials is for to know how much points every course worth.
The GPA is only a math fanction.